subsys/ieee802154/msglib/src/ahi_msg_lib.c

```diff
@@ -98,17 +98,89 @@ bool alif_ahi_msg_resp_event_recv(struct msg_buf *p_dest_msg, struct msg_buf *p_
 	return true;
 }
 
+bool alif_ahi_msg_recv_ind_recv_1_1_0(struct msg_buf *p_msg, uint16_t *p_ctx, int8_t *p_rssi,
```

Comment:
```suggestion
bool alif_ahi_msg_recv_ind_recv_1_2_0(struct msg_buf *p_msg, uint16_t *p_ctx, int8_t *p_rssi,
```

Reply:
Applies to all changes which mention v1.1

Reply:
It is at rom aldeary 1_1_0. So lets keep current names